Apps/WebClient/src/ClientApp/src/views/landing.vue

```diff
@@ -562,7 +448,6 @@ export default class LandingView extends Vue {
     ];
 
     private logo: string = Image00;
-    private introBackground: string = Image01;
```

Comment:
if the image is not being used let's remove it from the assets as well

Reply:
+1

Reply:
removed